src/main/kotlin/org/malachite/estella/commons/models/people/Organization.kt

```diff
@@ -8,5 +8,6 @@ import javax.persistence.*
 data class Organization(
         @Id @GeneratedValue(strategy = GenerationType.AUTO) val id: UUID?,
         val name: String,
-        val verified: Boolean?
+        @OneToOne val user: User,
+        val verified: Boolean? = false
```

Comment:
Zastanawia mnie, czemu verified jest nullable - w końcu zawsze jest albo false albo true - jaki jest use case nulla tutaj? W sensie co musiałoby się stać, żeby to było nullem?

Reply:
Było ustawione nullable, tak legacy więc nie zmieniłem, ale miałem się o to chyba pytać XD